core/services/runs.go

```diff
@@ -18,7 +18,7 @@ func MeetsMinimumPayment(
 	expectedMinJobPayment *assets.Link,
```

Comment:
Suggestion: Should we change this from needing a `*assets.Link` to just requiring a value, to be consistent? The comment below checking for `== nil` is still technically needed, unless we can guarantee a nil is never passed in.

Reply:
Yeah I'll see if this one can be changed as well.

Reply:
This is really the same one as below, I'd prefer to pass large structures in as pointers as this is the generally accepted go convention.